lib/cloudregister/registerutils.py

```diff
@@ -45,7 +45,7 @@
 REGISTRATION_DATA_DIR = '/var/cache/cloudregister/'
 REGISTERED_SMT_SERVER_DATA_FILE_NAME = 'currentSMTInfo.obj'
 RMT_AS_SCC_PROXY_MARKER = 'rmt_is_scc_proxy'
-
+DOCKER_REGISTRY_CREDENTIALS_PATH = '.docker/config.json'
```

Comment:
You can keep it that way but just to mention ```~/.docker/config.json``` is read if it exists by podman but is only searched for compat reasons for compatibility with docker. If not present the default location for the login credentials is

```
$XDG_RUNTIME_DIR/containers/auth.json
```

Reply:
I'm not sure I understand, from the podman doc

```
Podman first searches for the username and password in the ${XDG_RUNTIME_DIR}/containers/auth.json, if they are not valid, Podman then uses any existing credentials found in $HOME/.docker/config.json
```
so as I understand it, it will try 
```
$XDG_RUNTIME_DIR/containers/auth.json
```
and it will fail then it will try docker (which we are setting) so it will work

Reply:
Meaning setting only ~/.docker/config.json is enough for both podman and docker

Reply:
yes it will work as long as podman keeps compatible with docker in this way. As I said you can keep it the way it is. If you want to be super accurate you write both. By the way there are more container engines that supports OCI containers  e.g containerd (k3s/kubernetes) which does it different when using ```ctr image import ...```. Thus I don't think a generic handling is possible and that means explicit is better than implicit. But all that can be added later, just keep in mind this path is not common for everyone

Reply:
OK, AFAIK we are supporting docker and podman
That said, I will update the PR to set it explicitly for podman, in case compatibility changes in the future